### PR TITLE
API definition changed

### PR DESCRIPTION
**Pull request by yzainee-zz**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -66,21 +66,32 @@ paths:
...
+          required: true
+          type: string
+        - name: origin
+          in: header
+          description: This is required to give an indication as to from where is the request getting generated. If from VScode extension, the value which we need to send is 'vscode'.
+          required: false
+          type: string
+        - name: show_transitive
+          in: header
+          description: This is required to enable or disable the transitive support. We need to pass the value as True if transitive is needed. Default value is False.
```
Comment (tisnik) [inline swagger.yaml]: is it case independent please? Also please put 'True'/'False' into apostrophes or quotes, as these values are strings

File: swagger.yaml [hunk 2]
```diff
@@ -66,21 +66,32 @@ paths:
...
+          required: false
```
Comment (tisnik) [inline swagger.yaml]: is it required?

**Review by tisnik — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (yzainee) [reply swagger.yaml]: Yes this is case sensitive. We need to send 'True'

File: swagger.yaml (hunk 2 above)
Comment (yzainee) [reply swagger.yaml]: By default it takes false as the value. so not required

**Review by tisnik — APPROVED** [review]

**Pull request merged**